src/Vocup/Properties/Messages.nl.resx

```diff
@@ -259,6 +259,13 @@ Details: {0}</value>
   <data name="UnexpectedErrorT" xml:space="preserve">
     <value>Onbekende fout</value>
   </data>
+  <data name="VhfCompatMode" xml:space="preserve">
+    <value>Het woordenboekbestand werd opgeslagen door een latere versie van Vocup. Het zal geopend worden in compatibiliteitsmodus. Niet alle informatie zal zichtbaar zijn.

```

Comment:
Het woordenboekbestand is opgeslagen in een nieuwere versie van Vocup. Het zal geopend worden in compatibiliteitsmodus. Het kan zijn dat niet alle informatie zichtbaar is.

Reply:
Thanks for your suggestion.